.github/workflows/qc.yml

```diff
@@ -22,6 +22,9 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
+      - name: Install poetry dependencies
+        run: python -m pip install -U pip setuptools
```

Comment:
I assume @hrshdhgd has approved this; I don't really know exactly what may and may not go into the qc.yml file.

Reply:
I had not seen this before but this won't do any harm as far as I can tell.

Reply:
On the next PR, this continued failing, so this change didn't do anything :/